populate/src/main/resources/seed/iam/roles.json

```diff
@@ -12,6 +12,14 @@
       "participant_data_view"
     ]
   },
+  {
+    "name": "publisher",
+    "displayName": "Study publisher",
+    "description": "publish changes to irb and live environments",
+    "permissionNames": [
+      "publish"
+    ]
```

Comment:
I'm curious, why is this split out into a new role and not a part of `study_admin`? To me, study admin feels like the type of role that should be able to do _everything_ related to administering a study.

Or is the goal with this to treat this as a bit of a feature-gate since we're expanding this from superuser-only to non-superusers and we don't quite want to make the full jump yet?

Reply:
exactly -- we're still not ready for, e.g., OurHealth to own their own publishing.  So we want this separate for now.  In a few months we'll fold this into study_admin